internal/snapshot/save.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2022 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package snapshot
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flowkit"
+	"github.com/onflow/flow-cli/pkg/flowkit/services"
+)
+
+var SaveCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "save",
+		Short:   "Get the latest finalized protocol snapshot",
+		Example: "flow snapshot save /tmp/snapshot.json",
+	},
+	Flags: &struct {}{},
```

Comment:
Should the network flag be available here? deciding on which network you are connecting?

Reply:
--network part of the global flags